src/Operation/Delete.php

```diff
@@ -180,10 +180,6 @@ public function getCommandDocument(Server $server)
     {
         $cmd = ['delete' => $this->collectionName, 'deletes' => [['q' => $this->filter] + $this->createDeleteOptions()]];
 
-        if (isset($this->options['writeConcern'])) {
```

Comment:
I'm not sure if `readConcern` is relevant for delete operations.

Reply:
The `Delete` operation doesn't take a `readConcern` option, as the command doesn't support it either. If that was a typo and you meant `writeConcern`, then yes, it is relevant for delete operations as it is for all other writing commands, but it is not relevant when used in `explain`.

Reply:
Ah well, yes. That was not a typo. I didn't even think to check if the operation supported readConcern. So I'm doing the right thing by deleting these lines of code.

Reply:
Yes, that's correct.